_build_cfg.yml

```diff
@@ -332,6 +332,8 @@ Dir: dev_guide
 Topics:
   - Name: Overview
     File: index
+  - Name: The Application Life Cycle
```

Comment:
@bfallonf WDYT about dropping "The" here? And/or adding an "s": "Application Life Cycles".